modules/ossm-config-individual-control-plane-infrastructure-node-cli.adoc

```diff
@@ -3,13 +3,18 @@
 // * service_mesh/v2x/ossm-deployment-models.adoc
 
 :_content-type: PROCEDURE
-[id="ossm-config-individual-control-plane-infrastructure-node_{context}"]
-= Configuring individual {SMProductShortName} control plane components to run on infrastructure nodes
+[id="ossm-config-individual-control-plane-infrastructure-node-cli_{context}"]
+= Configuring individual control plane components to run on infrastructure nodes using the CLI
 
-This task should only be performed if individual {SMProductShortName} control plane components (such as Istiod, the Ingress Gateway, and the Egress Gateway) will run on infrastructure nodes. 
+Perform this task if individual components deployed by the {SMProductShortName} control plane, such as Istiod, the Ingress Gateway, and the Egress Gateway, will run on infrastructure nodes. 
 
 If the control plane will run on a worker node, skip this task.
 
+.Prerequisites
+
+* You have installed the {SMProductName} Operator.
+* You must have an account with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
```

Comment:
Same question as in my previous comment.

Reply:
done